tests/e2e-playwright/README.md

```diff
@@ -0,0 +1,145 @@
+# Sensei e2e tests
+
+## Quickstart
+You should first make sure that the [dependencies are installed](#dependencies). 
+Run `npm run test:e2e`.
+
+## Running
```

Comment:
I think it is worth adding a step with `wp-env start`.

Reply:
Good catch. 
Added!